### PR TITLE
[1.8] [Bugfix] Made armor stands send an EntityInteractEvent when right clicked on.

### PR DESCRIPTION
**Pull request by xandayn**
Armor stands aren't interacted with like other entities and a special function (currently named func_174825_a) is called when they're right clicked on, I've added a patch for the EntityArmorStand class that sends out a new EntityInteractEvent, and allows the event to be cancelled, when they're right clicked.

### CODE REVIEW

**Comment by LexManos** [comment]
This should be expanded to INTERACT_AT and pass in the vector information just like the C02PacketUseEntity was expanded.
Also, Sample mod!

**Comment by xandayn** [comment]
Could you maybe explain in simpler terms? I'm not too familiar with MinecraftForge and only noticed this because I was trying to use it in a mod I was making but was unable to get armor stands to send any events out. I'll make a Gist containing a sample mod right now though.

**Comment by LexManos** [comment]
Um no, my terms were pretty simple. Look at the classes I referenced and see how the functions are called.

**Comment by xandayn** [comment]
Oh okay I see, I misread the question, my apologies I didn't understand what you meant by expanding to INTERACT_AT but I get what you meant now, my mistake.

**Comment by TBlueF** [comment]
Is this still a thing? Because i'd need that Event ^^

**Comment by williewillus** [comment]
Closing as there is an INTERACT_AT version of the entity interact event in 1.8.x, and my Player Interact Event PR for 1.9 covers this as well

**Comment by Actuarius** [comment]
@williewillus closed this issue
